deploy/Dockerfile

```diff
@@ -4,29 +4,37 @@
 FROM ubuntu:22.04 AS base
 ENV DEBIAN_FRONTEND noninteractive
 RUN apt update --fix-missing && apt install \
-     # pkg-config is required for cargo to find libssl
-     libssl-dev pkg-config \
-     # rdkafka dependency needs cmake and a CXX compiler
-     cmake build-essential \
-     # rdkafka dependency needs libsasl \
-     libsasl2-dev \
-     # To install rust
-     curl  \
-     # For running the SQL compiler
-     openjdk-19-jre-headless -y \
-     # Install locale-gen
-     locales
+  # pkg-config is required for cargo to find libssl
+  libssl-dev pkg-config \
+  # rdkafka dependency needs cmake and a CXX compiler
+  cmake build-essential \
+  # rdkafka dependency needs libsasl \
+  libsasl2-dev \
+  # To install rust
+  curl  \
+  # For running the SQL compiler
+  openjdk-19-jre-headless -y \
+  # Install locale-gen
+  locales \
+  # To add the nodesource debian repository
+  ca-certificates gnupg
+
 
 # Set UTF-8 locale. Needed for the Rust compiler to handle Unicode column names.
 RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
-    locale-gen
+  locale-gen
 ENV LC_ALL en_US.UTF-8
 ENV LANG en_US.UTF-8
 ENV LANGUAGE en_US:en
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --profile minimal
-# web-console build tools
-RUN curl -fsSL https://deb.nodesource.com/setup_19.x | bash -
-RUN apt-get install --yes nodejs
+# web-console build tools:
```

Comment:
Do the web-console build tools need to be part of the base image? Note that the compiler-server derives from the base image, and I don't think it needs all this in it.

Reply:
I don't know but I just change the currently deprecated installation method. The node installation happened here previously. If we need to move it lets do one-by-one I can open a 2nd PR for it?

Reply:
If I understand correctly you suggest to move this here? https://github.com/feldera/feldera/blob/main/deploy/Dockerfile#L51

Reply:
Yes it can be a 2nd PR. And yes, moving it to the point you indicated should work.